src/main/java/com/lgtm/easymoney/controllers/GroupController.java

```diff
@@ -0,0 +1,43 @@
+package com.lgtm.easymoney.controllers;
+
+import com.lgtm.easymoney.models.Group;
+import com.lgtm.easymoney.payload.*;
+import com.lgtm.easymoney.services.GroupService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+@RestController
+@RequestMapping("/group")
+public class GroupController {
+    final GroupService groupService;
+
+    @Autowired
+    public GroupController(GroupService groupService) {
+        this.groupService = groupService;
+    }
+
+    @PostMapping("/create")
+    public ResponseEntity<ResourceCreatedRsp> createAGroup(@Valid @RequestBody CreateGroupReq createGroupReq) {
+        return groupService.createAGroup(createGroupReq);
+    }
+
+    @PutMapping("/invite")
```

Comment:
Since invite and leave only pass in ids, I wonder if we can pass ids as path variables and params so that we don't need these many `inviteGroupReq` and `leavGroupReq`. 🤔

Reply:
It might not be safe to show ids in path (as others can guess who is possibly in that group)

Reply:
By the way, once we have auth, we can use `@CurrentUser` and pass one less id

Reply:
I see this makes sense